ngrinder-controller/src/main/java/org/ngrinder/agent/controller/AgentDownloadController.java

```diff
@@ -73,11 +74,20 @@ public void download(@PathVariable String fileName, HttpServletResponse response
 	 * @param request request.
 	 */
 	@GetMapping("/{region}/{owner}")
-	public String downloadDirect(@PathVariable(value = "owner") String owner,
-	                             @PathVariable(value = "region") String region,
+	public String downloadDirect(@PathVariable String owner,
+	                             @PathVariable String region,
```

Comment:
switch the param order to follow their apearance.

Reply:
Fixed all.